tests/network-tests/src/misc/postMigrationAssertionsFlow.ts

```diff
@@ -0,0 +1,50 @@
+import { assert } from 'chai'
+import { FlowProps } from '../Flow'
+import { extendDebug } from '../Debugger'
+
+export default async function postMigrationAssertions({ api }: FlowProps): Promise<void> {
+    const debug = extendDebug('flow:postMigrationAssertions')
+    debug('Started')
+
+    debug('Ensure migration is done')
+
+    const channel_migration = await api.query.content.channelMigration();
+    const video_migration = await api.query.content.videoMigration();
+
+    assert(channel_migration.current_id.toNumber() == channel_migration.final_id.toNumber())
```

Comment:
```suggestion
    assert.equal(channel_migration.current_id.toNumber(), channel_migration.final_id.toNumber())
```
Better to use assert.equal to compare values when the assertion fails